sysusers.d/basic.conf.in

```diff
@@ -33,6 +33,7 @@ g lp      -     -            -
 g render  -     -            -
 g tape    -     -            -
 g video   -     -            -
+g sev     -     -            -
```

Comment:
this paragraph so far was ordered alphabetically...

Reply:
Fixed.